data/RI/incentive_relationships.json

```diff
@@ -9,6 +9,21 @@
     ],
     "RI-43": [
       "RI-13"
+    ],
+    "RI-45": [
+      "RI-46"
+    ],
+    "RI-47": [
+      "RI-48"
+    ],
```

Comment:
These are interesting. Because the pairs (45-46 and 47-48) are identical in all the filtering fields including low income, the excluded ones (46 and 48) will just never appear for anyone.

I think the thing to do is to consolidate each pair into a single incentive, and note the difference caused by the electrical-upgrade aspect in the description. Like the combination of 45 and 46 could be:

```
Rebate of 50% of cost for a Level 2 EV charger, up to $350, or up to $700 if an electrical upgrade is needed.
```

(And have 700 as the `maximum` in the amount)

What do you think?